pkg/controllers/v1/mpi_job_controller.go

```diff
@@ -1324,6 +1324,19 @@ func (c *MPIJobController) newLauncher(mpiJob *kubeflow.MPIJob, kubectlDeliveryI
 				},
 			},
 		})
+	
+	backOffLimit := new(int32)
+	*backOffLimit = 6
+	var activeDeadlineSeconds *int64
+	activeDeadlineSeconds = nil
```

Comment:
By default, `activeDeadlineSeconds` will be `nil`.

Reply:
Thanks. Removed this unnecessary assignment.